src/sync/borrowed.rs

```diff
@@ -164,6 +164,19 @@ impl RedisClientRemote {
     Box::new(commands_ft)
   }
 
+  /// Flush and close the Sender channel this instance receives messages through.
+  pub fn close(&mut self) {
+    let mut tx_guard = self.command_tx.write();
+    let mut tx_ref = tx_guard.deref_mut();
+
+    match *tx_ref {
```

Comment:
just to keep things consistent, can you do an `if let` here instead of a `match` with an empty block for the `None` variant

Reply:
For sure, I'll update the PR in a bit and let you know when it's ready.

Reply:
PR updated.